internal/server/config.go

```diff
@@ -32,12 +32,13 @@ type Jobs struct {
 }
 
 type Config struct {
-	Port                   int              `mapstructure:"port" default:"8080"`
-	GRPC                   GRPCConfig       `mapstructure:"grpc"`
-	EncryptionSecretKeyKey string           `mapstructure:"encryption_secret_key"`
-	Notifier               notifiers.Config `mapstructure:"notifier"`
-	LogLevel               string           `mapstructure:"log_level" default:"info"`
-	DB                     store.Config     `mapstructure:"db"`
+	Port                   int                         `mapstructure:"port" default:"8080"`
+	GRPC                   GRPCConfig                  `mapstructure:"grpc"`
+	EncryptionSecretKeyKey string                      `mapstructure:"encryption_secret_key"`
+	Notifier               notifiers.Config            `mapstructure:"notifier"`
```

Comment:
let's add deprecation mark for `Notifier`
```suggestion
  // Deprecated: use Notifiers instead
	Notifier               notifiers.Config            `mapstructure:"notifier"`
```